configs/mbaasy.json

```diff
@@ -11,11 +11,11 @@
     "lvl3": ".main .article h4",
     "lvl4": ".main .article h5",
     "lvl5": ".main .article h6",
-    "text": ".main .article p, .main .article li"
+    "text": ".main .article p, .main .article li, .main .article td, .main .article code"
```

Comment:
We do not recommend to index code as it introduced a lot of noise and algolia [does remove some specific character](https://www.algolia.com/doc/api-reference/api-parameters/separatorsToIndex/).

Please add a specific class like `.title` `.value` to the `td` element you want to index. There is a relation between them and it can be used as an upper `lvl`. Not every `td` are relevant and can be seen as noise

Reply:
hi @s-pace i've removed the `code` selector, however the `td` elements contain a lot of relevant content, see https://docs.mbaasy.com/client_api/apple_app_store/ as an example. The table headers are defined by a `th` so they won't get indexed. The only irrelevant column I see on this page is the "Required" column and this I can sort out by placing an icon instead.

Reply:
Ok, I will add a `first-child` selector too

Reply:
b34ccaf